pkg/kubelet/rkt/rkt.go

```diff
@@ -1581,18 +1755,42 @@ func (r *Runtime) GarbageCollect(gcPolicy kubecontainer.ContainerGCPolicy) error
 	return errors.NewAggregate(errlist)
 }
 
+func (r *Runtime) cleanupPodNetworkFromServiceFile(serviceFilePath string) error {
+	id, name, namespace, err := podDetailsFromServiceFile(serviceFilePath)
+	if err != nil {
+		return fmt.Errorf("rkt: Failed to read pod details from systemd service file %v: %v", serviceFilePath, err)
+	}
+	
+	r.cleanupPodNetwork(&api.Pod{
```

Comment:
btw, we cleanup pod network both in KillPod() and GC(), so maybe the error can be ignored?